wolfcrypt/src/wc_pkcs11.c

```diff
@@ -553,6 +572,25 @@ int wc_Pkcs11StoreKey(Pkcs11Token* token, int type, int clear, void* key)
                 break;
             }
     #endif
+    #if !defined(NO_AES) && defined(HAVE_AES_CBC)
+            case PKCS11_KEY_TYPE_AES_CBC: {
+                Aes* aes = (Aes*)key;
+
+                ret = Pkcs11MechAvail(&session, CKM_AES_CBC);
+                if (ret == 0) {
+                    ret = Pkcs11CreateSecretKey(&privKey, &session, CKK_AES,
+                                                (unsigned char *)aes->devKey,
+                                                aes->keylen,
+                                                (unsigned char *)aes->id,
+                                                aes->idLen);
+                }
+                if (ret == 0 && clear) {
+                    XMEMSET(aes->devKey, 0, aes->keylen);
```

Comment:
Should we be using ForceZero here instead?

Reply:
The aes.c `wc_AesFree` does a ForceZero on devKey... 
https://github.com/wolfSSL/wolfssl/blob/master/wolfcrypt/src/aes.c#L6598

Reply:
Fixed.